sessionctx/context.go

```diff
@@ -48,7 +49,12 @@ type Context interface {
 	NewTxn(context.Context) error
 	// NewStaleTxnWithStartTS initializes a staleness transaction with the given StartTS.
 	NewStaleTxnWithStartTS(ctx context.Context, startTS uint64) error
-
+	// SetDiskFullOpt set the disk full opt when tikv disk full happened.
+	SetDiskFullOpt(level kvrpcpb.DiskFullOpt)
+	// RollbackTxn rolls back the current transaction.
+	RollbackTxn(ctx context.Context)
+	// CommitTxn commits the current transaction.
+	CommitTxn(ctx context.Context) error
```

Comment:
Please add the reason for adding these functions here to the PR description.

Reply:
it already has some functions related to txn like `StmtCommit` , `StmtRollback`, and it very naturally adds `rollbackTxn` etc.

Reply:
What about the `SetDiskFullOpt`?

Reply:
the `session.context` will be used in ddl job operation, and only general ddls allowed to be executed when TiKV is disk full. So we will use `SetDiskFullOpt` method to control it